drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -571,6 +585,15 @@ local function initialize_switch(driver, device)
     device:set_field(COMPONENT_TO_ENDPOINT_MAP_BUTTON, component_map, {persist = true})
   end
 
+  -- If there is a custom static profile for the device, configure buttons if needed and
```

Comment:
Since the inovelli switch already has a sub driver, I would say we should remove this override from the main driver and just add the special handling for the `init` event in the inovelli sub driver. This way we can try to keep the base driver as generic as possible (although there are already exceptions to this).

So, I would just copy whatever functionality is needed from the base driver and then put it in the sub driver for the init function, and then trim whatever is not needed. Better yet, you could move the `configure_buttons` function to its own file so that it can be shared between the base and sub driver, similar to what is done with the `embedded-cluster-utils` module.